### PR TITLE
Apply feedback after Windows testing session

### PR DESCRIPTION
**Pull request by basvandriel**

### CODE REVIEW

**Comment by basvandriel** [comment]
Closes https://github.com/minvws/nl-uzipoc-yubisign/issues/39

File: app/appacme.py [hunk 1]
```diff
@@ -32,7 +32,7 @@ def __init__(
         Create an account. As per acme standard an email needs
         to be provided.
         """
-        areq = {"termsOfServiceAgreed": True, "contact": ["mailto:email@example.com"]}
+        areq = {"termsOfServiceAgreed": True, "contact": ["mailto:helpdesk@irealisatie.nl"]}
```
Comment (ricklambrechts) [inline app/appacme.py:35]: Would be nice if this is also configurable but for now this is fine as we are having more fixed values.

File: app/wizard.py [hunk 2]
```diff
@@ -54,7 +57,10 @@ def __init__(self, mypkcs, myacme, oidc_provider_base_url: urllib.parse.ParseRes
 
 
 if __name__ == "__main__":
-    load_dotenv()
+    env_file_path = PROJECT_ROOT / ".env"
+    load_dotenv(dotenv_path=env_file_path)
```
Comment (ricklambrechts) [inline app/wizard.py:61]: This should not be needed when inside the root folder and executing the program with `python -m app.wizard`.

File: docs/LOCALSETUP.md [hunk 3]
```diff
@@ -1,6 +1,10 @@
 # Local setup
+First clone this repository and checkout the `uzipoc_q4_2024` branch. You can then follow this document to set up and run the application.
+
+```bash
+git clone --single-branch --branch uzipoc_q4_2024 https://github.com/minvws/nl-uzipoc-yubisign
```
Comment (ricklambrechts) [inline docs/LOCALSETUP.md:5]:
```suggestion
git clone --single-branch --branch uzipoc_q4_2024 git@github.com:minvws/nl-uzipoc-yubisign.git
```

File: app/appacme.py (hunk 1 above)
Comment (basvandriel) [reply app/appacme.py:35]: I'll create a ticket for this

Comment (basvandriel) [reply app/appacme.py:35]: https://github.com/minvws/nl-uzipoc-yubisign/issues/41

File: app/wizard.py [hunk 4]
```diff
@@ -23,7 +24,9 @@
 import urllib.parse
 from dotenv import load_dotenv
 
-DEFAULT_ACME_CA_SERVER_URL = "https://acme.proeftuin.uzi-online.irealisatie.nl/directory"
+PROJECT_ROOT = Path(__file__).parent.parent
```
Comment (ricklambrechts) [inline app/wizard.py:27]: This is unused now.

**Pull request merged**